Support/Fuego/Mechanism/Models/PPHYS_CONSTANTS.H

```diff
@@ -0,0 +1,18 @@
+#ifndef _PPHYS_CONSTANTS_H_
+#define _PPHYS_CONSTANTS_H_
+
+ 
+
+#define PP_RU_CGS   83144626.1815324
+#define PP_RU_MKS   8.31446261815324
+#define PP_PA_CGS   1013250.0
+#define PP_PA_MKS   101325.0
+#define PP_RUC_CGS  1.98721558317399615845
+
+#define C_AW   12.011150 
+#define H_AW   1.007970
+#define O_AW   15.999400
+#define N_AW   14.006700
+#define AR_AW  39.948000
```

Comment:
We may want to just include all atomic weights.  See below, although a std::map apparently is not allowed in CUDA or DPC++ (very annoying), We can think about the best way to incorporate these:
    std::map<std::string,amrex::Real> AtomicWeight;
    AtomicWeight["H"] = 1.00797;
    AtomicWeight["HE"] = 4.00260;
    AtomicWeight["LI"] = 6.93900;
    AtomicWeight["BE"] = 9.01220;
    AtomicWeight["B"] = 10.81100;
    AtomicWeight["C"] = 12.01115;
    AtomicWeight["N"] = 14.00670;
    AtomicWeight["O"] = 15.99940;
    AtomicWeight["F"] = 18.99840;
    AtomicWeight["NE"] = 20.18300;
    AtomicWeight["NA"] = 22.98980;
    AtomicWeight["MG"] = 24.31200;
    AtomicWeight["AL"] = 26.98150;
    AtomicWeight["SI"] = 28.08600;
    AtomicWeight["P"] = 30.97380;
    AtomicWeight["S"] = 32.06400;
    AtomicWeight["CL"] = 35.45300;
    AtomicWeight["AR"] = 39.94800;
    AtomicWeight["K"] = 39.10200;
    AtomicWeight["CA"] = 40.08000;
    AtomicWeight["SC"] = 44.95600;
    AtomicWeight["TI"] = 47.90000;
    AtomicWeight["V"] = 50.94200;
    AtomicWeight["CR"] = 51.99600;
    AtomicWeight["MN"] = 54.93800;
    AtomicWeight["FE"] = 55.84700;
    AtomicWeight["CO"] = 58.93320;
    AtomicWeight["NI"] = 58.71000;
    AtomicWeight["CU"] = 63.54000;
    AtomicWeight["ZN"] = 65.37000;
    AtomicWeight["GA"] = 69.72000;
    AtomicWeight["GE"] = 72.59000;
    AtomicWeight["AS"] = 74.92160;
    AtomicWeight["SE"] = 78.96000;
    AtomicWeight["BR"] = 79.90090;
    AtomicWeight["KR"] = 83.80000;
    AtomicWeight["RB"] = 85.47000;
    AtomicWeight["SR"] = 87.62000;
    AtomicWeight["Y"] = 88.90500;
    AtomicWeight["ZR"] = 91.22000;
    AtomicWeight["NB"] = 92.90600;
    AtomicWeight["MO"] = 95.94000;
    AtomicWeight["TC"] = 99.00000;
    AtomicWeight["RU"] = 101.07000;
    AtomicWeight["RH"] = 102.90500;
    AtomicWeight["PD"] = 106.40000;
    AtomicWeight["AG"] = 107.87000;
    AtomicWeight["CD"] = 112.40000;
    AtomicWeight["IN"] = 114.82000;
    AtomicWeight["SN"] = 118.69000;
    AtomicWeight["SB"] = 121.75000;
    AtomicWeight["TE"] = 127.60000;
    AtomicWeight["I"] = 126.90440;
    AtomicWeight["XE"] = 131.30000;
    AtomicWeight["CS"] = 132.90500;
    AtomicWeight["BA"] = 137.34000;
    AtomicWeight["LA"] = 138.91000;
    AtomicWeight["CE"] = 140.12000;
    AtomicWeight["PR"] = 140.90700;
    AtomicWeight["ND"] = 144.24000;
    AtomicWeight["PM"] = 145.00000;
    AtomicWeight["SM"] = 150.35000;
    AtomicWeight["EU"] = 151.96000;
    AtomicWeight["GD"] = 157.25000;
    AtomicWeight["TB"] = 158.92400;
    AtomicWeight["DY"] = 162.50000;
    AtomicWeight["HO"] = 164.93000;
    AtomicWeight["ER"] = 167.26000;
    AtomicWeight["TM"] = 168.93400;
    AtomicWeight["YB"] = 173.04000;
    AtomicWeight["LU"] = 174.99700;
    AtomicWeight["HF"] = 178.49000;
    AtomicWeight["TA"] = 180.94800;
    AtomicWeight["W"] = 183.85000;
    AtomicWeight["RE"] = 186.20000;
    AtomicWeight["OS"] = 190.20000;
    AtomicWeight["IR"] = 192.20000;
    AtomicWeight["PT"] = 195.09000;
    AtomicWeight["AU"] = 196.96700;
    AtomicWeight["HG"] = 200.59000;
    AtomicWeight["TL"] = 204.37000;
    AtomicWeight["PB"] = 207.19000;
    AtomicWeight["BI"] = 208.98000;
    AtomicWeight["PO"] = 210.00000;
    AtomicWeight["AT"] = 210.00000;
    AtomicWeight["RN"] = 222.00000;
    AtomicWeight["FR"] = 223.00000;
    AtomicWeight["RA"] = 226.00000;
    AtomicWeight["AC"] = 227.00000;
    AtomicWeight["TH"] = 232.03800;
    AtomicWeight["PA"] = 231.00000;
    AtomicWeight["U"] = 238.03000;
    AtomicWeight["NP"] = 237.00000;
    AtomicWeight["PU"] = 242.00000;
    AtomicWeight["AM"] = 243.00000;
    AtomicWeight["CM"] = 247.00000;
    AtomicWeight["BK"] = 249.00000;
    AtomicWeight["CF"] = 251.00000;
    AtomicWeight["ES"] = 254.00000;
    AtomicWeight["FM"] = 253.00000;
    AtomicWeight["D"] = 002.01410;
    AtomicWeight["E"] = 5.48578E-4;  // Electrons

Reply:
While in theory yes we can include all these elements I think we should start with those we actually use now. We can always refine later.

Reply:
Yes, that's always the answer, it seems.  In the meantime, we should be able to detect if access is attempted for one that does not happen to be in the magic HONA set, and the code aborts with an informative message rather than stepping out of bounds in some array in the bowels of the physics routines.

Reply:
I am not sure it is used in the way you think it is. Right now those atomic weights are not used. I placed them there to be able to, in the future, perhaps to simplify these mixture frac things we were talking about. So plenty of time to make it better ! 
So yes what you offer is in my mind a bit ahead of what we are trying to do. BTW right now, is there one mech except electron stuff that uses something else than CHONAR ?

Reply:
I don't think that's the best way to make these decisions. The list is finite and already assembled.  If there is a way to easily include all cases so that it never has to be the source of a core dump later, then at the cost of a few extra lines of easy-to-read code, it should be done.